components/layouts/default.js

```diff
@@ -10,7 +10,6 @@ export const Main = styled.main`
   @media ${breakpoints.tabletAndAbove} {
     grid-row: 1/4;
     grid-column: 2;
-    max-width: 55ch;
```

Comment:
<img width="1820" alt="image" src="https://github.com/medienhaus/medienhaus-spaces/assets/706419/1ad77d6c-44f2-4acb-a94c-0d737dcdbd7b">
<img width="1820" alt="image" src="https://github.com/medienhaus/medienhaus-spaces/assets/706419/4005643a-57fd-467b-a24f-9082d3a218fc">

Reply:
i will just @andirueckel here 🙃

Reply:
The highlighted `max-width`  was removed from the “default layout” on purpose.

We should not, by default, limit the width of our `main` element; this *was* done in another prototype with very different layout and structure. If we want to limit the widths of the sections shown in the above screenshots, then we should do so in their respective components. I’d argue, though, that we might want to adapt the layout logic/structure per se as we made many changes to [medienhaus-spaces](https://github.com/medienhaus/medienhaus-spaces).

Reply:
I'm cool with either way. I'm just not cool with how the `/login` page is looking now. Feel free to implement whatever you prefer, or bring back the `max-width` for now and propose a more general layout change in a separate PR.